### ISSUE TITLE
Can this work be combined with multi-resolution hash coding?

### ISSUE THREAD

**Issue by BCL123456-BAL** [Jiawei-Yang/FreeNeRF#4]
It's a great job! 
But I noticed that your results were obtained on 4 NVIDIA A40 GPUs. 
This is very counterintuitive, less mirrors of the training view will increase the burden of training. 
Can it be combined with hash multi-resolution coding to speed up training time and reduce GPU burden？

Best wish
Chenglong

**Comment by SophusDavid** [comment]
I try to reproduce this work(mainly freq_mask) on nerfstudio and here is my code [https://github.com/SophusDavid/FreeNeRF_NeRFStudio](url)
But actually the performance is bad.Any advice is welcomed!

**Comment by Jiawei-Yang** [comment]
Hi there, 

Thanks for your attention to our work! We do think combining the idea of FreeNeRF with multi-grid / hash-grid-based methods will be a promising direction for future work! 

@SophusDavid Thank you for your initial implementation! It would be great if you can further elaborate on the bad performance, e.g., what the bad results look like. Did you try to monitor the change of frequency mask as a sanity check? I'm currently out of bandwidth to go through the entire implementation, but I will definitely do so when I'm available!

**Comment by SophusDavid** [comment]
> Hi there,
> 
> Thanks for your attention to our work! We do think combining the idea of FreeNeRF with multi-grid / hash-grid-based methods will be a promising direction for future work!
> 
> @SophusDavid Thank you for your initial implementation! It would be great if you can further elaborate on the bad performance, e.g., what the bad results look like. Did you try to monitor the change of frequency mask as a sanity check? I'm currently out of bandwidth to go through the entire implementation, but I will definitely do so when I'm available!

Sorry,that was a mistake . I just checked my code and there are some bugs. When I fixed them,the result seems similar to the paper.I'm going to test metrics tomorrow or the day after tomorrow.By the way ,can you tell me which images you use for training? I tried to find in the paper but failed.Thanks for your awesome work again :)

**Comment by gucka28** [comment]
Hello, @Jiawei-Yang

Thank you for sharing your work!
Could you share the result of your implementation on benchmark dataset (LLFF or Blender dataset)?
Does the same frequency masking hyper-parameter works good in Hash-grid?
